Sources/Orbit/Components/HorizontalScroll.swift

```diff
@@ -0,0 +1,235 @@
+import SwiftUI
+
+public enum HorizontalScrollItemWidth {
+    /// Each item will use its own intrinsic width.
+    case intrinsic
+    /// Width ratio compared to container width. Default value is `0.8`.
```

Comment:
> Default value is `0.8`.
Isn't this visible in quick help these days? I'd avoid restating it here because the comment can go out of sync with the implementation.

Reply:
✔️